apps/vr-tests-react-components/src/stories/ShadowDOM/ShadowDOMDefault.stories.tsx

```diff
@@ -0,0 +1,45 @@
+import { makeStyles, shorthands } from '@griffel/react';
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 3 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=28395&commitId=4af114e5b1d5883dfb6aed5108ea45bc05e96158&env=prod)

✅ There was 3 screenshots added, 0 screenshots removed, 2002 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Shadow DOM.Default.default.chromium.png|0| Added|
|Shadow DOM.Portal.chromium.png|0| Added|
|Shadow DOM.Portal.normal.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv9"/>